src/asmcnc/production/z_head_qc_jig/z_head_qc_7.py

```diff
@@ -116,11 +116,11 @@ def resume_from_alarm(self):
         self.m.resume_from_alarm()
 
     def do_cycle(self):
-        self.m.s.write_command('G53 G0 Z-150')
+        self.m.s.write_command('G53 G0 Z-' + str(self.m.grbl_z_max_travel))
         self.m.s.write_command('G53 G0 Z-1')
-        self.m.s.write_command('G53 G0 Z-150')
+        self.m.s.write_command('G53 G0 Z-' + str(self.m.grbl_z_max_travel))
         self.m.s.write_command('G53 G0 Z-1')
-        self.m.s.write_command('G53 G0 Z-150')
+        self.m.s.write_command('G53 G0 Z-' + str(self.m.grbl_z_max_travel))
```

Comment:
This movement seems quite common, a function with parameter for Z could be valuable.

Reply:
@archiepeterjarvis I agree, but will make a new ticket for it :)